vacs-map-app/src/components/BaseMap.vue

```diff
@@ -47,6 +47,10 @@ onMounted(() => {
   if (mapPadding.value) map.value.setPadding(mapPadding.value)
 })
 
+onUnmounted(() => {
+  map.value.remove();
+});
```

Comment:
This was a memory leak--Mapbox GL was holding onto resources even though we've navigated away from the map.